oak_remote_attestation/proto/v1/BUILD

```diff
@@ -19,6 +19,7 @@ load("@io_grpc_grpc_java//:java_grpc_library.bzl", "java_grpc_library")
 load("@rules_cc//cc:defs.bzl", "cc_proto_library")
 load("@rules_java//java:defs.bzl", "java_proto_library")
 load("@rules_proto//proto:defs.bzl", "proto_library")
+load("//dart:build_defs.bzl", "dart_proto_library")
```

Comment:
Where is it defined? In the `third_party`?

Reply:
Not sure. I am still experimenting to figure out how to load `dart_proto_library`.

Reply:
I cannot find anything dart-related in https://github.com/stackb/rules_proto. Any ideas how to fix this @tiziano88?